events_webhook/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2017 Adobe. All rights reserved.
+    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License. You may obtain a copy
+    of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under
+    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+    OF ANY KIND, either express or implied. See the License for the specific language
+    governing permissions and limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>com.adobe.aio</groupId>
+    <artifactId>aio-lib-java</artifactId>
+    <version>1.0.7-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <name>Adobe I/O - Events Webhook Library</name>
+  <description>Adobe I/O - Java SDK - Events Webhook Library</description>
+  <properties>
+    <junit.jupiter.version>5.3.1</junit.jupiter.version>
+    <wiremock.version>2.32.0</wiremock.version>
+  </properties>
+  <artifactId>aio-lib-java-events-webhook</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.adobe.aio</groupId>
+      <artifactId>aio-lib-java-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <!--- test specific -->
+    <dependency>
```

Comment:
woudn't be `junit:junit` as specified in the parent project `dependencyManagement` enough ?

Reply:
its for `junit-jupiter` , specifying all three is required else getting errors during tests build

Reply:
no only the below dependencies are needed 

```
  <dependencies>
    <dependency>
      <groupId>com.adobe.aio</groupId>
      <artifactId>aio-lib-java-core</artifactId>
      <version>${project.version}</version>
    </dependency>

    <!--- test specific -->
    <dependency>
      <groupId>junit</groupId>
      <artifactId>junit</artifactId>
    </dependency>
    <dependency>
      <groupId>com.github.tomakehurst</groupId>
      <artifactId>wiremock-jre8</artifactId>
    </dependency>
  </dependencies>
```

just rewrite your test to be consistent with the other test of aio-lib-java and change your jupiter imports with 

```
import org.junit.After;
import org.junit.Assert;
import org.junit.Before;
import org.junit.Test;

```